README.md

```diff
@@ -108,13 +109,63 @@ If you’re starting totally new, we got you! Here’s a 101 conceptual read on
 
 Find nice articles with rich and valuable content about different topics related to Filecoin network.
 
-## Looking for a JS library to interact with Filecoin? :wrench::hammer:
+## Looking for the complete documentation? :books::books:
+
+Filecoin solidity documentation: [Let's go to docs web](https://docs.filecoin.io/smart-contracts/developing-contracts/solidity-libraries/) :arrow_upper_right:
 
-We have been working hard to create a complete library that covers every aspect you may need in order to interact with 
-the Filecoin blockchain. [Let's go to library](https://www.npmjs.com/package/@zondax/izari-filecoin) :arrow_upper_right:
+## Getting Started
 
-## Looking for the complete documentation? :books::books:
+Requirements / Steps are performed on MacOS.
+
+### Requirements:
+
+-   **Foundry** [[Official Docs]](https://book.getfoundry.sh/getting-started/installation):
+
+    `curl -L https://foundry.paradigm.xyz | bash`
+
+-   **Rust** [[Official Docs]](https://doc.rust-lang.org/book/ch01-01-installation.html):
+
+    `curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf | sh`
+
+-   **Yarn**[[Official Docs]](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable):
+
+    `npm install --global yarn`
+
+-   **CMake** [[Official Docs]](https://cmake.org/download/):
```

Comment:
The deliverables for Filecoin.sol won't need to take foundry or rust as dependencies will they? A lot of FEVM developers are using Hardhat. I imagine as an NPM package import it won't matter for the consumer?